src/components/SearchInput.vue

```diff
@@ -193,6 +193,12 @@ export default {
   margin-bottom: 0;
 }
 
```

Comment:
IE displays its own widget for clearing the input that overlaps with the one we provide.